### ISSUE TITLE
Re-open: Inconsistent documentation locations, docs vs website/docs

### ISSUE THREAD

**Issue by AliAllomani** [F5Networks/terraform-provider-bigip#518]
Re-opening issue #515

@RavinderReddyF5 Thanks for the update, would it be possible to advise what was the fix for this ?

from what i can see, the mentioned missing params has been added to `website/docs/r/bigip_ltm_profile_server_ssl.html.markdown`

but i think there should be more long-term solution for this, as having 2 separately managed directories makes confusion to the contributors, and there might be several resources that were updated in one location but not the other.

for example, all the data sources exist in `/website/docs` but not `/docs`

ref. : https://www.terraform.io/docs/registry/providers/docs.html#migrating-legacy-providers-docs

**Comment by RavinderReddyF5** [comment]
@AliAllomani For Now Hashicorp supporting documentation from both places, and i think it is picking up documentation from website directory if it exists, we are pushing all documentation into website/docs. we make sure all reflected in docs folder also.
as you mentioned if there are any discrepancy we will try to address the same. we will work on retiring one of directory in upcoming releases.

**Comment by AliAllomani** [comment]
Current diffs

```bash
for f in $(ls website/docs/r); do diff -qr  docs/resources/${f/.html.markdown/.md} website/docs/r/${f}; done

Files docs/resources/bigip_as3.md and website/docs/r/bigip_as3.html.markdown differ
diff: docs/resources/bigip_bigiq_as3.md: No such file or directory
Files docs/resources/bigip_common_license_manage_bigiq.md and website/docs/r/bigip_common_license_manage_bigiq.html.markdown differ
diff: docs/resources/bigip_event_service_discovery.md: No such file or directory
Files docs/resources/bigip_fast_application.md and website/docs/r/bigip_fast_application.html.markdown differ
Files docs/resources/bigip_fast_template.md and website/docs/r/bigip_fast_template.html.markdown differ
Files docs/resources/bigip_ltm_node.md and website/docs/r/bigip_ltm_node.html.markdown differ
Files docs/resources/bigip_ltm_policy.md and website/docs/r/bigip_ltm_policy.html.markdown differ
Files docs/resources/bigip_ltm_pool.md and website/docs/r/bigip_ltm_pool.html.markdown differ
Files docs/resources/bigip_ltm_pool_attachment.md and website/docs/r/bigip_ltm_pool_attachment.html.markdown differ
Files docs/resources/bigip_ltm_profile_httpcompress.md and website/docs/r/bigip_ltm_profile_httpcompress.html.markdown differ
Files docs/resources/bigip_ltm_profile_oneconnect.md and website/docs/r/bigip_ltm_profile_oneconnect.html.markdown differ
Files docs/resources/bigip_ltm_profile_server_ssl.md and website/docs/r/bigip_ltm_profile_server_ssl.html.markdown differ
Files docs/resources/bigip_ltm_profile_tcp.md and website/docs/r/bigip_ltm_profile_tcp.html.markdown differ
Files docs/resources/bigip_ltm_virtual_address.md and website/docs/r/bigip_ltm_virtual_address.html.markdown differ
Files docs/resources/bigip_ltm_virtual_server.md and website/docs/r/bigip_ltm_virtual_server.html.markdown differ
Files docs/resources/bigip_sys_provision.md and website/docs/r/bigip_sys_provision.html.markdown differ
```

```bash
for f in $(ls website/docs/d); do diff -qr  docs/data-sources/${f/.html.markdown/.md} website/docs/d/${f}; done

diff: docs/data-sources/bigip_ltm_datagroup.md: No such file or directory
diff: docs/data-sources/bigip_ltm_irule.md: No such file or directory
diff: docs/data-sources/bigip_ltm_monitor.md: No such file or directory
diff: docs/data-sources/bigip_ltm_node.md: No such file or directory
diff: docs/data-sources/bigip_ltm_pool.markdown: No such file or directory
diff: docs/data-sources/bigip_ltm_ssl_certificate.md: No such file or directory
diff: docs/data-sources/bigip_vwan_config.md: No such file or directory
```

**Comment by RavinderReddyF5** [comment]
@AliAllomani thanks for your support in listing differences. we track it internally.

**Comment by trinaths** [comment]
Created INFRAANO-434 for internal tracking

**Comment by RavinderReddyF5** [comment]
@AliAllomani fixed in v1.11.0
